textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/TestConstants.java

```diff
@@ -65,4 +65,52 @@
         return Arrays.asList(tuple1, tuple2, tuple3, tuple4, tuple5);
                 
     }
+    
+    //sample created by team3

```

Comment:
The main project shouldn't mention "team_X" since the logic is independent from teams.

Since there can be many constants used for different domains, it will be better to have a separate file for the constants for each domain.  For example, you can use "TestCorpConstants.java".